packages/rest/src/http/route.ts

```diff
@@ -161,10 +162,14 @@ export function resolvePathAndParameters(
     }
   }
 
+  const shared =
```

Comment:
I would move this logic in the @route decorator not leave it to however is consuming it.

Reply:
Yep, I moved the logic in very closely to where the validation logic exists.